### PR TITLE
Refactor methods and fetchit config

### PR DESCRIPTION
**Pull request by sallyom**
Here it is
I updated the `examples/*.yaml` so you can check out the difference in the config file
You can now enter a list of each method in a single target.

### CODE REVIEW

**Comment by cooktheryan** [comment]
Almost got it. I feel like after this is merged maybe we should cut a 0.0.1 release that way if we do change the configuration we can cut a new release so we don't break people's environment

File: examples/config-url.yaml [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
 # Note: It takes (default) 5 min for raw urls to be updated in GitHub, so this test will wait for 7 min
-targets:
-- name: fetchit
-  url: http://github.com/redhat-et/fetchit
-  methods:
-    configTarget:
-      configUrl: https://raw.githubusercontent.com/josephsawaya/fetchit/better-apply/examples/config-reload.yaml
-      schedule: "*/1 * * * *"
-
+configTarget:
```
Comment (cooktheryan) [inline examples/config-url.yaml:3]: Maybe fetchitConfig?

File: docs/methods.rst [hunk 2]
```diff
@@ -104,22 +103,22 @@ look for image updates with all podman-generated unit files that include the aut
    volume: fetchit-volume
    targets:
```
Comment (cooktheryan) [inline docs/methods.rst:104]: Do all of these targets need to become targetConfigs?

File: examples/config-url.yaml (hunk 1 above)
Comment (sallyom) [reply examples/config-url.yaml:3]: or `fetchitConfigFile`

File: docs/methods.rst (hunk 2 above)
Comment (sallyom) [reply docs/methods.rst:104]: yes, I'll update that

Comment (sallyom) [reply docs/methods.rst:104]: updated - also brought in your commit

File: examples/config-url.yaml (hunk 1 above)
Comment (sallyom) [reply examples/config-url.yaml:3]: going with `ConfigReload` for now

**Comment by sallyom** [comment]
Greeeen ✌️

**Review by josephsawaya — APPROVED** [review]
lgtm

**Pull request merged**